### PR TITLE
Features/risk of change

### PR DESCRIPTION
**Pull request by Kinlaw**
Adds risk of change to actions, planner and system modal.

Also replaces ignore rule text with a toggle.

### CODE REVIEW

**Comment by Kinlaw** [comment]
On hold, found a bug in system modal where the resolution_risk is coming back as null. I think it's probably just a missing param in the api call.

File: app/js/states/rules/list-rules.jade [hunk 1]
```diff
@@ -34,22 +34,25 @@ section
...
-                      .sep
-                      rule-toggle(rule="rule")
+                  .flexgrid.equal-heights
+                    .row
+                      .col-sm-8
+                        .icon-group.icon-group-severity
+                          severity-icon(severity='rule.rec_impact', type='impact')
+                          severity-icon(severity='rule.rec_likelihood', type='likelihood')
+                          severity-icon(severity='rule.severity', type='severity')
+                        risk-of-change
```
[the remark this answers is not in the document]
Comment (Kinlaw) [reply app/js/states/rules/list-rules.jade]: Good catch @jharting

File: app/js/components/maintenance/maintenancePlan/maintenancePlan.jade [hunk 2]
```diff
@@ -238,18 +238,20 @@
...
+                      .col-md-4.resolution
                         h4.section-title(translate) Resolution:
                         p {{ play.ansible_resolutions[0].description }} &nbsp;
                           a(ng-if='play.ansible_resolutions.length > 1', ng-click='resolutionModal(play)', tooltip='{{"Multiple resolutions available. Click here to select a different one." | translate}}')
                             span(translate) - edit ({{play.ansible_resolutions.length}} options)
                         .icon-inline.push-top-narrow.red(ng-if='play.ansible_resolutions[0].needs_reboot')
                           i.fa.fa-repeat
                           strong(translate, tooltip = "{{'System reboot is required to resolve this issue' | translate}}") Reboot Required
+                      .col-md-3
+                        risk-of-change(change-risk='play.ansible_resolutions[0].resolution_risk')
```
[the remark this answers is not in the document]
Comment (Kinlaw) [reply app/js/components/maintenance/maintenancePlan/maintenancePlan.jade]: I think this is a good point. @mattnolting I'd like to get your thoughts on placement of the riskOfChange component.

**Comment by mattnolting** [comment]
I added some css and markup to support inline layout.

**Comment by jharting** [comment]
@mattnolting 
screenshot_20170927_093924

are the lines supposed to be aligned?

**Comment by mattnolting** [comment]
@jharting yes, they are. updating now

**Pull request merged**